docs/content/docs/build/containers.md

```diff
@@ -107,72 +114,48 @@ const { container, services } =
     await client.getContainer(/*container id*/, schema);
```

Comment:
We are using container IDs in our telemetry to represent instance of the container runtime. Clients will often use our telemetry to log/observer errors so this will certainly create confusion. At the very least we likely need more explicit clarification on telemetry/logging docs.